graph/import.go

```diff
@@ -13,13 +13,19 @@ import (
 	"github.com/docker/docker/utils"
 )
 
+// ImageImportConfig holds configuration to import a image.
 type ImageImportConfig struct {
-	Changes         []string
-	InConfig        io.ReadCloser
-	OutStream       io.Writer
+	//Chages are the container changes writtent to top layer
+	Changes []string
+	//InConfig is the input stream containers layered data
+	InConfig io.ReadCloser
+	//OutStream is the output stream where the image is written
+	OutStream io.Writer
+	//ContainerConfig is the configuration of commit container
 	ContainerConfig *runconfig.Config
 }
 
+// Import allows to download image from  a archive.
```

Comment:
s/from  a/from an/

I think this comment should be *completed*. ``Import`` looks at ``src`` and if ``src`` is ``-`` it will use ``InConfig`` which is a reader that comes from ``stdin`` (as documented it's possible to do ``cat exampleimage.tgz | docker import - exampleimagelocal:new``). Your comment should mention that depending on the specified source ``src`` and the ``ImageImportConfig`` it will read an archive from the provide ``InConfig`` reader or will download the archive and read from it. Also it should mention that it's not only downloading the image but create the layer  with the specified repo/tag :wink:.